src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -121,123 +80,27 @@ const CreateCacheWizard = (props) => {
     const [stepIdReached, setStepIdReached] = useState(1);
 
     // State for the form (Getting Started)
-    const [gettingStarted, setGettingStarted] = useState<GettingStarted>(GettingStartedInitialState);
+    const [gettingStarted, setGettingStarted] = useState<GettingStartedState>(GettingStartedInitialState);
 
     // State for the form (Edit Code)
-    const [cacheEditor, setCacheEditor] = useState<CacheEditor>(CacheEditorInitialState);
+    const [cacheEditor, setCacheEditor] = useState<CacheEditorState>(CacheEditorInitialState);
 
     // State for the form (Configuration Basic)
     const [basicConfiguration, setBasicConfiguration] = useState<BasicConfiguration>(BasicConfigurationInitialState);
 
     // State for the form (Advanced Options)
-    const [advancedOptions, setAdvancedOptions] = useState<AdvancedOptions>(AdvancedOptionsInitialState);
+    const [advancedOptions, setAdvancedOptions] = useState<AdvancedConfiguration>(AdvancedOptionsInitialState);
 
-    // State for the form (Review)
-    const [reviewOptions, setReviewOptions] = useState<ReviewOptions>(ReviewOptionsInitialState);
+    const [configuration, setConfiguration] = useState<CacheConfiguration>({basic: basicConfiguration, advanced:advancedOptions})
 
     const history = useHistory();
 
-    useEffect(() => {
-        ConsoleServices.dataContainer()
-            .getCacheConfigurationTemplates(cmName)
-            .then((eitherTemplates) => {
-                if (eitherTemplates.isRight()) {
-                    let options: OptionSelect[] = [];
-                    eitherTemplates.value.forEach((template) => {
-                        options.push({ value: template.name });
-                    });
-                    cacheEditor.configs = options;
-                } else {
-                    addAlert(eitherTemplates.value);
-                }
-            })
-    }, []);
-
     const title = 'Create Cache';
 
-    const createCache = () => {
```

Comment:
@karesti this function was working for creating cache with templates also. Aren't we going to use templates for cache creation in future also?